saltgui/static/stylesheets/page.css

```diff
@@ -101,6 +100,14 @@ table tr td:last-of-type {
   padding-right: 0;
 }
 
+table tr td.starttime,
+table tr td.target {
+  white-space: pre-wrap;
+  max-width: 20%;
+  width: 20%;
```

Comment:
`width` should not be neccesary?

Reply:
Without `width` container is always shrinked to the minimal width and wrapping is applied. That's why I've added this.

Reply:
When you use `white-space: nowrap`, then `width` and `max-width` can be removed.

Reply:
`width` and `max-width` removed.